src/main/resources/templates/experiments.html

```diff
@@ -105,7 +105,7 @@ <h4>Info</h4>
                                                                     <tr>
                                                                         <td th:text="${nodesDetailsMap.key}"></td>
                                                                         <td th:text="${nodesDetailsMap.value.os}"></td>
-                                                                        <td th:text="${nodesDetailsMap.value.qualifiedName}"></td>
+                                                                        <td><a th:href="@{/web_ssh/access_node/{qualifiedName}(qualifiedName = ${nodeDetailsMap.value.qualifiedName})}" th:text="${nodesDetailsMap.value.qualifiedName}"></a></td>
```

Comment:
(qualifiedName = ${nodeDetailsMap.value.qualifiedName})} should be (qualifiedName = ${node**s**DetailsMap.value.qualifiedName})}

It is "nodes" and not "node".